tests/ctl/core/test_api.py

```diff
@@ -2057,11 +2057,15 @@ def test_system_update_privacy_declaration_cookies(
         # System level cookies removed
         assert result.json()["cookies"] == []
         # Privacy declaration cookies added
-        assert sorted(result.json()["privacy_declarations"][0]["cookies"], key=lambda r: r["name"]) == sorted([
+        assert sorted(
+            result.json()["privacy_declarations"][0]["cookies"], key=lambda r: r["name"]
+        ) == sorted(
+            [
                 {"name": "my_cookie", "path": None, "domain": "example.com"},
                 {"name": "my_other_cookie", "path": None, "domain": None},
-            ], key=lambda r: r["name"])
-
+            ],
+            key=lambda r: r["name"],
+        )
```

Comment:
these changes seem to have been introduced when running the `static_checks` command